code/modules/client/preferences.dm

```diff
@@ -2021,7 +2021,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	var/datum/loadout_category/DLC = GLOB.loadout_categories["Donator"] // stands for donator loadout category but the other def for DLC works too xD
 	if(!LAZYLEN(GLOB.patrons) || !CONFIG_GET(flag/donator_items)) // donator items are only accesibile by servers with a patreon
 		return
-	if(IS_PATRON(parent.ckey))
+	if(IS_PATRON(parent.ckey) || (parent in GLOB.admins) || (parent in GLOB.mentors))
```

Comment:
```suggestion
	if(IS_PATRON(parent.ckey) || (parent in GLOB.admins))
```

Not in the headspace to actually add configs and deal with all that, so for now I'll just do admins and revisit mentors later.